equipment_handler/include/sTransmitter.h

```diff
@@ -78,6 +81,7 @@ typedef enum {
     BAD_READ = 1,
     BAD_WRITE = 1,
     BAD_PARAM = 2,
+    IS_STUBBED = 0,
```

Comment:
This is indistinguishable from the FUNC_PASS, is that intentional?

Reply:
Yes so that higher level application code can use logic that checks if an S-band function returned 0 (FUNC_PASS) and not throw an error if the S-band is stubbed. Arash originally did this with the UHF code.